example_test.go

```diff
@@ -14,18 +14,18 @@ import (
 // should cause PAM to ask its conversation handler for a username and password
 // in sequence.
 func Example() {
-	t, err := pam.StartFunc("", "", func(s pam.Style, msg string) (string, error) {
+	t, err := pam.StartFunc("passwd", "", func(s pam.Style, msg string) (string, error) {
 		switch s {
 		case pam.PromptEchoOff:
-			fmt.Print(msg)
+			fmt.Printf("%s", msg)
```

Comment:
```suggestion
			fmt.Printf("%s: ", msg)
```

Reply:
Yeah, definitely... It's up to the module to define those..

It's quite weird that it does assumptions on the UI is running though, since a module could message can be print anywhere (including a non-terminal UI), so it's not supposed to send that.

Reply:
Adding the space is fine in any case though... Since in the password case we won't  see anything printing anyways.